compose/hubmap.stage.yml

```
@@ -0,0 +1,44 @@
+version: "3.7"
+
+services:
+
+  portal-ui:
+    hostname: portal-ui
+    container_name: portal-ui
+    image: hubmap/portal-ui:latest
+
+    # Mount the app config in container to keep it outside the image.
+    volumes:
+      - "../context/instance/app.conf:/app/instance/app.conf"
+
+    # Avoid accidentally creating zombie processes
+    init: true
+
+    # Specifying a restart policy to avoid downtime
+    restart: always
+
+    networks:
+      - gateway_hubmap
+
+  portal-ui-prod:
+    hostname: portal-ui-prod
+    container_name: portal-ui-prod
+    image: hubmap/portal-ui:latest
+
+    # Mount the app config in container to keep it outside the image.
+    volumes:
+      - "../context/instance/app.prod.conf:/app/instance/app.conf"
+
+    # Avoid accidentally creating zombie processes
+    init: true
+
+    # Specifying a restart policy to avoid downtime
+    restart: always
+
+    networks:
+      - gateway_hubmap
```

```suggestion
  portal-ui: &default
    hostname: portal-ui
    container_name: portal-ui
    image: hubmap/portal-ui:latest

    # Mount the app config in container to keep it outside the image.
    volumes:
      - "../context/instance/app.conf:/app/instance/app.conf"

    # Avoid accidentally creating zombie processes
    init: true

    # Specifying a restart policy to avoid downtime
    restart: always

    networks:
      - gateway_hubmap

  portal-ui-prod:
    <<: *default
    hostname: portal-ui-prod
    container_name: portal-ui-prod

    # Mount the app config in container to keep it outside the image.
    volumes:
      - "../context/instance/app.prod.conf:/app/instance/app.conf"
```
Perhaps use yaml syntax to highlight just the parts that are different?